sift/__main__.py

```diff
@@ -226,12 +226,202 @@ def _common_path_prefix(paths):
         return None
 
 
-class Main(QtGui.QMainWindow):
-    _last_open_dir = None  # directory to open files in
-    _recent_files_menu = None  # QMenu
+class UserControlsAnimation(QtCore.QObject):
+    """Controller behavior focused around animation bar and next/last key bindings.
+    Connects between document, scene graph manager, layer list view of document; uses UI elements.
```

Comment:
Could you add a blank line between the first "subject line" and the body of the docstring?

Reply:
done